.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,56 @@
+name: Qonsole Integration Tests
+
+on:
+  push:
+    branches: [ main, master ]
+    paths:
```

Comment:
Where did this pattern of adding paths here come from? Isn't it a bit of an anti-pattern that could lead to more WTF/minute?

Reply:
Again, as discussing in slack - I have added this to allow future updates - I want to leave this until we decide we need/want to change the primary branch name.